static/styles/recent_topics.css

```diff
@@ -120,6 +120,16 @@
                 background-color: hsl(0, 0%, 80%);
                 outline: 0;
             }
+
+            &.fake_disabled_button {
+                cursor: not-allowed;
+                opacity: 0.5;
```

Comment:
How did you select 0.5 for this? Ideally we'd have this styling be as similar to a `disable` style as we can.

Reply:
This is the same as bootstrap sets for elements with `disabled` attribute.